nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/test/java/org/apache/nifi/util/db/TestJdbcCommon.java

```diff
@@ -465,7 +474,12 @@ private void testConvertToAvroStreamForBigDecimal(BigDecimal bigDecimal, int dbP
             assertEquals("decimal", logicalType.getName());
             LogicalTypes.Decimal decimalType = (LogicalTypes.Decimal) logicalType;
             assertEquals(expectedPrecision, decimalType.getPrecision());
-            assertEquals(expectedScale, decimalType.getScale());
+            // If scale > precision then precision will be set to the scale value
+            if (expectedScale > expectedPrecision) {
```

Comment:
This conditional always evaluates to false when running the unit test in debug mode.  Does something need to be changed in the new unit test method?